### DISCUSSION TITLE
Does the mqtt net server support share subscription, which is the new feature in mqtt V5?

### DISCUSSION THREAD

**Discussion by liptonn** [dotnet/MQTTnet#1414] [Q&A] [answered]
Does the mqtt net server support share subscription, Is there any specific switch I need to turn on or what? 
I tried, but seems not working, if the answer is yes, please someone show me an example, and god bless you :D

**Comment by SeppPenner** [comment, marked as answer]
This is already answered in https://github.com/dotnet/MQTTnet/issues/1404. --> Not yet supported, low prio to be supported soon :)
